### PR TITLE
DCOS-48818: translate text

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
@DanielMSchmidt , file conflicts.

**Comment by GeorgiSTodorov** [comment]
Hold merge until the other PR is merged.

File: plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const MultiContainerFormAdvancedSection = ({ data, path }) => {
         Advanced Settings
       </Trans>
       <Trans render="p">Advanced settings of the container.</Trans>
-      <p>{getForcePullSection(data, path)}</p>
+      <div className="form-group">{getForcePullSection(data, path)}</div>
```
Comment (GeorgiSTodorov) [inline plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js]: Is this related to file-based secrets?

File: plugins/services/src/js/components/forms/NetworkingFormSection.js [hunk 2]
```diff
@@ -590,8 +593,12 @@ class NetworkingFormSection extends mixin(StoreMixin) {
 
     const selections = (
       <FieldSelect name="networks.0.network" value={selectedValue}>
-        <Trans render={<option value={HOST} />}>Host</Trans>
-        <Trans render={<option value={BRIDGE} />}>Bridge</Trans>
+        <Trans key="host" render={<option value={HOST} key="host" />}>
```
Comment (GeorgiSTodorov) [inline plugins/services/src/js/components/forms/NetworkingFormSection.js]: And this?

File: plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js (hunk 1 above)
Comment (DanielMSchmidt) [reply plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js]: Not at all, but CI is failing due to prettier changes not being conform, therefore I added those as a separate commit

Comment (DanielMSchmidt) [reply plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js]: Oh wait those are different ones, I am unsure how they landed here, let me check

Comment (DanielMSchmidt) [reply plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js]: Must have been a wrongly resolved merge conflict, I'll recreate the branch once everything is working properly

Comment (GeorgiSTodorov) [reply plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js]: Cool.

**Comment by GeorgiSTodorov** [comment]
Failing test seems unrelated, retriggering.

**Comment by GeorgiSTodorov** [comment]
It's green now.
